cmd/doctor.go

```diff
@@ -65,7 +66,16 @@ func Doctor() *cobra.Command {
 			}
 			dev, err := utils.GetDevFromManifest(manifest, devName)
 			if err != nil {
-				return err
+				if !errors.Is(err, oktetoErrors.ErrNoDevSelected) {
```

Comment:
We should probably also add `oktetoErrors.ErrManifestNoDevSection` to this if

Reply:
this if controls that if there is no dev selected, we show the options to select the dev to the user. If there is no dev section, the error will be returned and showed to the user. Do we want to prompt the init so the dev is created at the manifest?

Reply:
No, it's ok as it is. I read wrong the code and thought it was displaying the selector when the dev didn't exist sorry 😅